pkg/services/navtree/navtreeimpl/admin.go

```diff
@@ -135,7 +136,7 @@ func (s *ServiceImpl) getAdminNode(c *contextmodel.ReqContext) (*navtree.NavLink
 		configNodes = append(configNodes, storage)
 	}
 
-	if s.features.IsEnabled(ctx, featuremgmt.FlagOnPremToCloudMigrations) && c.SignedInUser.IsGrafanaAdmin {
+	if s.features.IsEnabled(ctx, featuremgmt.FlagOnPremToCloudMigrations) && c.SignedInUser.HasRole(org.RoleAdmin) {
```

Comment:
🤔 I get you're changing this to give any admin of each org permissions, right? Not clear in the issue description ;)

Reply:
You're right - I forgot to update the description.

Note, this isn't to properly finish that task we added today, this is just a temp-ish change.

Reply:
(updated pr description)

Reply:
I understand this is kind of the same as mocking the API. We will review these parts when we have a proper API, right?

Reply:
Added the missing and mandatory labels ;) so the tests can pass

Reply:
Sorry, not sure why my last comment has been duplicated 🤷

Reply:
When the actual APIs that can 'cause damage' are implemented, the backend developers will implement them with proper access control checks. These checks here are only for showing the UI in the nav tree, and not actually granting access to anything sensitive.